app/models/article.rb

```diff
@@ -634,16 +634,16 @@ def evaluate_front_matter(front_matter)
     self.collection_id = Collection.find_series(front_matter["series"], user).id if front_matter["series"].present?
   end
 
-  def determine_image(front_matter)
-    # In order to clear out the cover_image, we check for the key in the front_matter.
-    # If the key exists, we use the value from it (a url or `nil`).
-    # Otherwise, we fall back to the main_image on the article.
-    has_cover_image = front_matter.include?("cover_image")
-
-    if has_cover_image && (front_matter["cover_image"].present? || main_image)
-      front_matter["cover_image"]
+  def set_image(front_matter)
+    if front_matter.include?("cover_image")
```

Comment:
I checked whether `cover_image` appearing anywhere else in the front matter (for example, `description: Something something cover_image`) would inadvertently set this and it worked as intended. 👍🏼

Reply:
@jgaskins I think there's some antics with `.include?`.  The `front_matter` is a Hash, and `.include?` just checks the keys.  My recommended refactor is to move from `.include?` to `.key?`